api/tests/routes/native/security_test.py

```diff
@@ -0,0 +1,36 @@
+from unittest.mock import patch
+
+import pytest
+
+from pcapi.core.users import factories as users_factories
+from pcapi.models import api_errors
+from pcapi.routes.native.security import setup_context
+
+
+@pytest.mark.usefixtures("db_session")
+class SetupContextTest:
+    @patch("pcapi.routes.native.security.get_jwt_identity")
+    def test_get_active_user(self, mocked_get_jwt_identity):
+        user = users_factories.UserFactory(isActive=True)
+        mocked_get_jwt_identity.return_value = user.email
+
+        res = setup_context(lambda x: not x.isActive)
+        assert res.id == user.id
+
+    @patch("pcapi.routes.native.security.get_jwt_identity")
+    def test_get_unknown_user(self, mocked_get_jwt_identity):
+        # built but not saved in db: can't find any user with its email
+        # address
+        user = users_factories.UserFactory.build(isActive=True)
+        mocked_get_jwt_identity.return_value = user.email
+
+        with pytest.raises(api_errors.ForbiddenError):
+            setup_context(lambda x: not x or not x.isActive)
```

Comment:
toutes les données du test sont controllées en amont

je trouve ça dommage de mettre de la logique dans les tests

```suggestion
            setup_context(lambda _user: False)
```